lib/commands/general.js

```diff
@@ -116,6 +116,12 @@ commands.keys = async function (keys) {
     }
     await this.setValue(keys, el.ELEMENT);
   } else {
+    if (_.isArray(keys)) {
+      keys = keys.join('');
+    }
+    if (!_.isString(keys)) {
+      keys = `${keys}`;
```

Comment:
Ah! I see. Yes, `toString` is better. And not sure why I went with the other, to be perfectly honest.